changes/ee/feat-10778.en.md

```diff
@@ -0,0 +1 @@
+Refactored Pulsar Producer bridge to avoid leaking resources during crashes.
```

Comment:
```suggestion
Refactored Pulsar Producer bridge to avoid leaking resources during crashes at creation.
```

Reply:
fixed in https://github.com/emqx/emqx/pull/10813